src/main/java/com/regnosys/testing/testpack/TestPackConfigCreator.java

```diff
@@ -11,9 +13,15 @@ public interface TestPackConfigCreator {
     /**
      * Generates pipeline and test-pack config files.
      *
-     * @param rosettaPaths - list of folders that contain rosetta model files, e.g. "drr/rosetta"
-     * @param filter       - provides filters to include or exclude
-     * @param testPackDefs - provides list of test-pack information such as test pack name, input type and sample input paths
+     * @param rosettaPaths      - list of folders that contain rosetta model files, e.g. "drr/rosetta"
+     * @param filter            - provides filters to include or exclude
+     * @param testPackDefs      - provides list of test-pack information such as test pack name, input type and sample input paths
+     * @param functionSchemaMap - function / xsd look up map
+     * @param injector          - model runtime guice injector
      */
-    void createPipelineAndTestPackConfig(ImmutableList<String> rosettaPaths, TestPackFilter filter, List<TestPackDef> testPackDefs);
+    void createPipelineAndTestPackConfig(ImmutableList<String> rosettaPaths, 
+                                         TestPackFilter filter, 
+                                         List<TestPackDef> testPackDefs, 
+                                         ImmutableMap<Class<?>, String> functionSchemaMap, 
```

Comment:
Could you rename this to either `schemaMap` or `outputTypeSchemaMap`, and update the javadoc comment (and do the same in the implementing class

Reply:
yeah sure